app/code/Magento/User/Block/Role/Tab/Users.php

```diff
@@ -45,7 +45,9 @@ protected function _construct()
         $roleId = $this->getRequest()->getParam('rid', false);
         /** @var \Magento\User\Model\ResourceModel\User\Collection $users */
         $users = $this->_userCollectionFactory->create()->load();
-        $this->setTemplate('role/users.phtml')->assign('users', $users->getItems())->assign('roleId', $roleId);
+        $this->setTemplate('Magento_User::role/users.phtml')
```

Comment:
Please, revert fix in places when the template specified dynamically

Reply:
Done.